Menus/LoadingGUI.cpp

```diff
@@ -186,7 +186,12 @@ namespace RTE {
 					char outputDirName[s_MaxFileName];
 					char parentDirName[s_MaxFileName];
 					// Copy the file path to a separate directory path
+#ifdef __STDC_LIB_EXT1__
 					strcpy_s(outputDirName, sizeof(outputDirName), outputFileName);
+#else
+          // Some compilers dont yet implement strcpy_s use unsafe strcpy instead
```

Comment:
Check your spacing. I don't care about it in files that haven't been updated to proper formatting, but this has and you should maintain it.

Reply:
I've gone through all the changes again, I think I caught it all now